src/core/Table/Table.tsx

```diff
@@ -416,6 +439,24 @@ export const Table = <
 
   const areFiltersSet = allColumns.some((column) => !!column.filterValue);
 
+  const onRowClickHandler = React.useCallback(
+    (event: React.MouseEvent, row: Row<T>) => {
+      if (isSelectable) {
+        if (!row.isSelected && !event.ctrlKey) {
+          dispatch({
+            type: TableActions.singleRowSelected,
+            id: row.id,
+            value: true,
```

Comment:
Started to think that `value` might not be needed as it is always true.

Reply:
Removed